app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -58,6 +60,11 @@ def get_network_interfaces
         @network_interfaces = gather_data_for_this_region(@nis)
       end
 
+      # Can use any service object here, results are identical.
+      def get_subscriptions
+        @subscriptions = @vmm.subscriptions.map(&:subscription_id)
```

Comment:
@djberg96 technically it is better to take a snapshot (cache) of subscriptions at refresh time. We don't want to collect some resources for n subscriptions and other resources for n+1 subscriptions. In reality this is unlikely to happen because a new subscription is created with empty resources.